src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -123,12 +177,97 @@ export class SentryAppExternalIssueForm extends React.Component<Props> {
     }
   }
 
+  /**
+   * This function determines which fields need to be reset and new options fetched
+   * based on the dependencies defined with the depends_on attribute.
+   * This is a hack to get around the issue that the autoLoad parameter for the deprecated
+   * Select control does not actually get componn
+   */
+  handleFieldChange = async (id: string, _value: FieldValue) => {
```

Comment:
Since this isn't used...

```suggestion
  handleFieldChange = async (id: string) => {
```